documentation/modules/mtv-resources-and-services.adoc

```diff
@@ -28,14 +27,11 @@ Only one `Migration` CR per migration plan can run at a given time. You can crea
 ** Runs the `Validation` service if a VM configuration change is detected.
 
 * The `Validation` service checks the suitability of a VM for migration by applying rules.
-* The `User Interface` service performs the following actions:
-** Enables you to create and configure {project-short} CRs.
-** Displays the status of the CRs and the progress of a migration.
-
```

Comment:
shouldn't we keep a blank line between the validation service and the migration controller service in terms of formatting?

Reply:
@ahadas Not quite sure what you mean. Please take a look at preview #2, which I just added, and see if the layout looks OK to you.

Reply:
ack, it looks well in the preview